packages/nextra-theme-docs/src/components/toc.tsx

```diff
@@ -57,12 +57,18 @@ export function TOC({ toc, filePath }: TOCProps): ReactElement {
   return (
     <div
       className={cn(
-        'nextra-scrollbar _sticky _top-16 _overflow-y-auto _px-4 _pt-6 _text-sm [hyphens:auto]',
+        '_grid _grid-rows-[1fr_min-content]', // 1fr: scrollbar, min-content: toc footer
+        '_sticky _top-16 _pt-6 _text-sm [hyphens:auto]',
         '_max-h-[calc(100vh-var(--nextra-navbar-height)-env(safe-area-inset-bottom))]'
       )}
     >
       {hasHeadings && (
-        <>
+        <div
+          className={cn(
+            'nextra-scrollbar _overflow-y-auto _px-4',
```

Comment:
I see the benefit of applying  `nextra-scrollbar` to this `div` instead of a parent, this is nice!

## before

https://github.com/user-attachments/assets/aaf7fd91-c540-4ce4-a723-acc3b556e7c9


## after

https://github.com/user-attachments/assets/adf1fa51-bc5c-433e-967d-f6e1e00b0539